template/index.html

```diff
@@ -22,33 +22,33 @@
             <table width="100%" cellpadding="0" cellspacing="0" border="0">
                 <tbody><tr>
                     <td style="padding: 10px 20px 20px 20px;">
-                        UK Daily Email, <a class="story-title" href="/daily-email/v1" > v1</a> | <a class="story-title" href="/daily-email/v2015" > v2015</a> |
-                         <a class="story-title" href="/headline" >headline</a> |
+                        UK Daily Email, <a class="story-title" href="/daily-email/v1" > v1</a>{{ " " }}|{{ " " }}<a class="story-title" href="/daily-email/v2015" > v2015</a>{{ " " }}|{{ " " }}
```

Comment:
No, that's what you have to do if you want to force the minifier not to remove whitespace :/